src/DIPS.Xamarin.UI/Controls/Tag/Tag.xaml.cs

```diff
@@ -1,73 +1,112 @@
-using Xamarin.Forms;
+﻿using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
 
 namespace DIPS.Xamarin.UI.Controls.Tag
 {
     /// <summary>
     ///     A <c>Frame</c> wrapping a <c>Label</c> to allow setting border radius for the <c>Label</c>.
     /// </summary>
-    public class Tag : Frame
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class Tag : ContentView
     {
+        /// <inheritdoc />
+        public Tag()
+        {
+            InitializeComponent();
+        }
+
+        #region Label Properties
+
         /// <summary>
         ///     Bindable property for <see cref="Text" />
         /// </summary>
-        public static readonly BindableProperty TextProperty = Label.TextProperty;
+        public static readonly BindableProperty TextProperty =
+            BindableProperty.Create(nameof(Text), typeof(string), typeof(Tag), default(string));
 
         /// <summary>
         ///     Bindable property for <see cref="TextColor" />
         /// </summary>
-        public static readonly BindableProperty TextColorProperty = Label.TextColorProperty;
+        public static readonly BindableProperty TextColorProperty =
+            BindableProperty.Create(nameof(TextColor), typeof(Color), typeof(Tag), Color.Default);
 
         /// <summary>
         ///     Bindable property for <see cref="FontFamily" />
         /// </summary>
-        public static readonly BindableProperty FontFamilyProperty = Label.FontFamilyProperty;
+        public static readonly BindableProperty FontFamilyProperty =
+            BindableProperty.Create(nameof(FontFamily), typeof(string), typeof(Tag), default(string));
 
         /// <summary>
         ///     Bindable property for <see cref="FontAttributes" />
         /// </summary>
-        public static readonly BindableProperty FontAttributesProperty = Label.FontAttributesProperty;
+        public static readonly BindableProperty FontAttributesProperty = BindableProperty.Create(nameof(FontAttributes),
+            typeof(FontAttributes), typeof(Tag), FontAttributes.None);
 
         /// <summary>
         ///     Bindable property for <see cref="FontSize" />
         /// </summary>
         public static readonly BindableProperty FontSizeProperty =
-            BindableProperty.Create(nameof(FontSize), typeof(double), typeof(Tag), -1.0);
+            BindableProperty.Create(nameof(FontSize), typeof(double), typeof(Tag), -1d);
```

Comment:
FontSize should have a TypeConverter to enable consumers of using Named Font Sizes for accessability scaling. I did this for our Date and Time Picker, see [this](https://github.com/DIPSAS/DIPS.Xamarin.UI/blob/474b2f6fbef75b0a0d6d62e119bbffe5cd0a6110/src/DIPS.Xamarin.UI/Controls/TimePicker/TimePicker.xaml.cs#L78)

Reply:
Oh, I have missed my converter filter 🤯

Anyway, you don't need to create your own converter here, I used `Xamarin.Forms.FontSizeConverter`,

```
[TypeConverter(typeof(FontSizeConverter))]
public double FontSize { get { ... } set { ... } }
```